influunt-app/app/views/controladores/index.html

```diff
@@ -1,77 +1,83 @@
-  </div>
-  <div class="col-lg-10 col-sm-10">
-  <section role="complementary" class="">
-      <div class="col-lg-4 col-sm-4 no-padding">
-        <form role="form" class="form-group">
-          <div class="input-group">
-            <input type="text" class="form-control" ng-model="filtros.campo_busca">
-            <span class="input-group-btn">
-              <button type="button" data-toggle="tooltip" data-placement="top" title="Pesquisar" class="btn btn-primary">
-                <i class="fa fa-search" aria-hidden="true"></i>
-              </button>
-            </span>
           </div>
-        </form>
-      </div>
-      <!--
-      <div class="col-lg-2 col-sm-2">
-        <button type="button" class="btn btn-primary"><i class="fa fa-th-large" aria-hidden="true"></i></button>
-        <button type="button" class="btn btn-primary"><i class="fa fa-th-list" aria-hidden="true"></i></button>
-      </div>
-      -->
-      <div class="clearfix"></div>
-      <div class="col-lg-2 no-padding">
-        <button type="button" class="btn btn-primary container-fluid" data-toggle="tooltip" data-placement="top" title="Adicionar novo controlador" ui-sref="app.wizard_controladores.dados_basicos">Novo controlador</button>
-      </div>
-      <div class="clearfix"></div>
-      <div class="col-lg-12 controller no-padding">
-        <ul>
-          <li ng-repeat="controlador in lista | filter: filtros.campo_busca | checkboxFilter:filtroLateral">
-            <div class="ibox float-e-margins">
-              <div class="ibox-title">
-                <h5>{{ controlador.CLC }}</h5>
-              </div>
-              <div class="ibox-content">
-                <p class="title">{{ controlador.localizacao }}</p>
-                <!--
-                <p class="status"><b>Status:</b></p>
-                <p class="status-active"><i class="fa fa-circle text-navy"></i>Ativo</p>
-                -->
-                <hgroup>
-                  <a class="btn btn-primary" data-toggle="tooltip" data-placement="top" title="Editar controlador" type="button" ui-sref="app.wizard_controladores.dados_basicos({id: controlador.id})">
-                    <i class="fa fa-pencil"></i>
-                  </a>
-                  <a class="btn btn-primary" data-toggle="tooltip" data-placement="top" title="Vizualizar controlador" type="button" ui-sref="app.controladores_show({id: controlador.id})">
-                    <i class="fa fa-eye"></i>
-                  </a>
-                  <a class="btn btn-primary" data-toggle="tooltip" data-placement="top" title="Vizualizar planos" type="button" ui-sref="app.planos({id: controlador.id})">
-                    <i class="fa fa-file-text"></i>
-                  </a>
-                </hgroup>
+          <div class="ibox-content">
+            <div class="row">
+              <div class="col-sm-3 m-b-xs"><select class="input-sm form-control input-s-sm inline">
+                <option value="">Escolha a Área</option>
```

Comment:
Por enquanto as áreas vão ser fixas? Se for vamos criar uma issue para resolver isso depois.